discoverx/dx.py

```diff
@@ -49,7 +49,7 @@ def __init__(
 
     def _can_read_columns_table(self) -> bool:
         try:
-            self.spark.sql(f"SELECT * FROM {self.COLUMNS_TABLE_NAME} LIMIT 1")
+            self.spark.sql(f"SELECT * FROM {self.COLUMNS_TABLE_NAME} WHERE table_catalog = 'system' LIMIT 1")
```

Comment:
@souravg-db mentioned to maybe even add a filter on the table_schema. Might make sense but maybe doesn't reduce the query time much more?